src/custom/components/TransactionSettings/index.tsx

```diff
@@ -17,6 +18,15 @@ const Option = styled(FancyButtonUni)<{ active: boolean }>`
   }
 `
 
+const Wrapper = styled.div`
+  ${OptionCustom} {
```

Comment:
is it not possible to override OptionCustom with styled? by adding className prop should work no?

Reply:
Probably yes, but to me this looks the least obtrusive without adding stuff to the Mod file. Think I should modify?